src/content/enterprise/install/releases.mdx

```diff
@@ -5,6 +5,31 @@ sidebar_label: Release Notes
 id: releases
 ---
 
+## 0.14.0
+TBD
+
+### Features
+- Add structured and staged logs for pipelines. Groups logs by stage in the okteto UI.
+- Support custom envvars for main okteto components
+- Support defining custom annotation, labels and envvars in pipeline installer jobs
+- Display CLI version for pipeline installer jobs in the Okteto UI admin panel
+- Add ability to download the Okteto cluster context from the UI
+
+### Improvements
+- Upgrade Okteto CLI to [2.4.1](https://github.com/okteto/okteto/releases/tag/2.4.1)
+- Add better defaults for timestamps in event logs
+- Remove deprecated Google Sign-in Javascript Platform Library in favor or standard OAuth. More info [here](https://developers.googleblog.com/2021/08/gsi-jsweb-deprecation.html)
+- Remove the [helm catalog](https://www.okteto.com/docs/enterprise/administration/configuration/#applications) on self-hosted installations by default. This makes installing CRDs not mandatory during cluster creation.
+
+
+### Bugfixes
+- Improve backend api upgrades
+- Do not attempt to redeploy snapshots in the webhook for update operations
+- Account for installer jobs token in registry auth
```

Comment:
What does this mean?

Reply:
This is https://github.com/okteto/app/pull/4077

@pchico83 how can we put this?

Reply:
@rberrelleza the token we were using in installer jobs wasn't supported in our registry auth layer. this was breaking the use case to use our helm registry as a hemp chart repository

Reply:
can we phrase it that way then? The way is described today doesn't tell me anything as a user or admin of okteto.